BootloaderCommonPkg/Library/ElfLib/ElfLib.inf

```diff
@@ -22,8 +22,18 @@
 [Sources]
   ElfCommon.h
   Elf32.h
+  Elf64.h
+  ElfLibInternal.h
   ElfLib.c
 
+[Sources.IA32]

```

Comment:
Do you think we can make it simpler to just use a same for both IA32 and X64 ?
If required,  IS_X64 can be used to determine x86 or x64 build.  The compiler will remove unused code if size is the concern.

Reply:
Sure, let me make it common for both IA32 and X64.